### PR TITLE
feat: fetch app docs_url

### PR DESCRIPTION
**Pull request by chipytux**
## Pull Request Description

**Title:** Fetch docs_url dynamically from FastAPI app

**Description:**

This pull request introduces a change to the `fastapi-cli` library that fetches the `docs_url` value dynamically from the FastAPI application instead of using a hardcoded value. 

**Implementation Details:**

The change involves modifying the `_run` function to fetch the `docs_url` value from the FastAPI application using the `app.openapi_url` attribute. This ensures that the library always uses the URL specified by the application itself.

**Testing:**

The updated function has been thoroughly tested to ensure it correctly retrieves the `docs_url` value from the FastAPI application and returns the appropriate URL.

### CODE REVIEW

**Review by pedroimpulcetto — APPROVED** [review]

File: src/fastapi_cli/cli.py [hunk 1]
```diff
@@ -45,6 +48,13 @@ def callback(
     """
 
 
+def _get_docs_url(uvicorn_path: str) -> str:
+    module_path, app_name = uvicorn_path.split(sep=":")
+    module = importlib.import_module(module_path)
+    app = getattr(module, app_name)
+    return app.docs_url or DEFAULT_DOCS_URL
```
Comment (killjoy1221) [inline src/fastapi_cli/cli.py:55]: Doesn't `docs_url is None` mean the docs are disabled.

Comment (chipytux) [reply src/fastapi_cli/cli.py:55]: I couldn't figure out the difference between the `docs_url` field not being informed or this same field being set to `None`. I ended up changing it only for cases where the value was informed, otherwise I kept the same value that was hardcoded.

Comment (chipytux) [reply src/fastapi_cli/cli.py]: @killjoy1221 I modified the code to omit the `Api doc` section if `docs_url=None` is explicitly specified. Please review.

**Comment by chipytux** [comment]
Do you think this is a good approach @tiangolo ? 🙏🏻

😃 @pedroimpulcetto

**Review by pedroimpulcetto — APPROVED** [review]

**Comment by chipytux** [comment]
Do you think this is a good approach @tiangolo ? 🙏🏻

**Comment by chipytux** [comment]
@tiangolo I do not have permission to configure a label for the PR. https://github.com/fastapi/fastapi-cli/actions/runs/10408815023/job/28827030912?pr=24

**Comment by tobb10001** [comment]
I ran across the same bug today and I can confirm that the provided patch resolves the issue.

**Comment by pedroimpulcetto** [comment]
oh, ok, I'm able to handle these changes since @chipytux isn't aware of it

**Comment by YuriiMotov** [comment]
I meant we need to resolve the merge conflicts (merge `main` branch into the branch of this PR) and ensure it still works as expected.

The `entrypoint` option was added recently, we need to make sure it works correctly with these changes.
And, we need to add tests with edge cases like `root_path`.
